cmd/cmd.go

```diff
@@ -39,6 +39,10 @@ var (
 func CommandLine(driver, targetDb, projectID, instanceID, dbName string, dataOnly, schemaOnly, skipForeignKeys bool, schemaSampleSize int64, sessionJSON string, ioHelper *conversion.IOStreams, outputFilePrefix string, now time.Time) error {
 	var conv *internal.Conv
 	var err error
+	targetDb, isNewDb, err := conversion.VerifyTargetDb(targetDb, projectID, instanceID, dbName)
```

Comment:
why not move this block inside the createOrGetDatabase  ?

Reply:
I put this check before so that for existing DBs, we do the schema conversion with the correct TargetDB value. Moving it there would mean identifying targetDB after schema conv is completed. Also for cases, when user has a DB with some DDL in it, I feel it is better to fail fast rather than after schema conv is done.